archive/archive_api/src/responses.py

```diff
@@ -35,23 +33,12 @@ def __init__(self, response, *args, **kwargs):
         # The @context may already be provided if we've been through the
         # force_type method below.  We also don't add a context if we're
         # looking at the healthcheck endpoint.
-        if ("@context" in rv) or (rv == {"status": "OK"}):
-            return super().__init__(response, *args, **kwargs)
-        else:
-            rv["@context"] = self.context_url
-            json_string = json.dumps(rv)
-            return super().__init__(json_string, *args, **kwargs)
+        return super().__init__(response, *args, **kwargs)
```

Comment:
At this point I suspect we can ditch this entire class.

Reply:
I think so but I couldn't figure out how to do that and I couldn't be bothered anyway cause all this code is going away entirely. We still have very out of date models in the models folder for example